### DISCUSSION TITLE
Command "toktx" not found?

### DISCUSSION THREAD

**Discussion by luyangtop** [KhronosGroup/KTX-Software#539] [Q&A] [answered]
cmd：
gltf-transform etc1s gg.glb paddle2.glb
then error:
error: Command "toktx" not found. Please install KTX-Software, from:

https://github.com/KhronosGroup/KTX-Software

how to install KTX-Software in win10?
I have download and install KTX-Software-4.0.0-win64.exe,but the error still have.

**Comment by luyangtop** [comment]
Not global cmd，should cmd in the toktx.exe path.How can cmd in global?

**Reply by MarkCallow** [reply, marked as answer]
The directory of `toktx` needs to be in your %Path% environment variable. It should have been added when you installed `KTX-Software-4.0.0-win64.exe` but I regularly get an error telling me the installer can't add `toktx`'s directory because the %Path% is too long. In my case this appears bogus as my %PATH% is nowhere near 2048 bytes but I have not been able to find a workaround. When this happens, add the directory manually. The default location is `c:\Program Files\KTX-Software\bin`.

**Reply by ignarmezh** [reply]
I also geet this error, and I settled manually PATH to Environment Variables.
I can't figure out what the problem could be.
I installed version 4.2.1. And forexample standard commands work "gltf-transform optimize input.glb output.glb --compress draco --texture-compress webp".

Edit.: I transfered from global Environment variables to local (only for one user) and reboot pc, it works now.

**Reply by MarkCallow** [reply]
We have fixed the issue with setting the PATH during installation. It was an issue with NSIS (NullSoft Install System) not handling long strings.

Whether `KTX-Software/bin` is added to the global PATH or user PATH should not make any difference. Anyway I'm glad you got it working.
